c/pthread-nondet/Makefile

```diff
@@ -0,0 +1,10 @@
+LEVEL := ../
+
+COMMON_WARNINGS := \
+	-Wno-int-conversion \
+
+CLANG_WARNINGS := \
+	-Wno-incompatible-library-redeclaration \
+	-Wno-return-type \
```

Comment:
Only ignoring `-Wno-return-type` is necessary for the tasks in this directory. The other two warnings do not actually need to be ignored.